cli/src/pcluster/aws/aws_resources.py

```diff
@@ -249,9 +249,9 @@ def name(self) -> str:
         return self._image_data.get("Name")
 
     @property
-    def original_image_name(self) -> str:
-        """Return original image name without date."""
-        return self._get_tag(PCLUSTER_IMAGE_NAME_TAG)
+    def build_image_id(self) -> str:
```

Comment:
If this is the class with the EC2 Image information, I would call this `pcluster_image_id`